clients/src/main/java/org/apache/kafka/clients/consumer/internals/events/ListOffsetsApplicationEvent.java

```diff
@@ -85,9 +85,10 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return getClass().getSimpleName() + " {" +
-                "timestampsToSearch=" + timestampsToSearch + ", " +
-                "requireTimestamps=" + requireTimestamps + '}';
+        return "ListOffsetsApplicationEvent{" +
```

Comment:
Is it intentional to explicitly include "ListOffsetsApplicationEvent" here? it is in the "owner" section of the toStringBase too.

Reply:
(Similar for all other events that are explicitly including the class name and then calling the toStringBase)